astroquery/mpc/core.py

```diff
@@ -1088,24 +1094,27 @@ def _parse_result(self, result, **kwargs):
                 elif 's=s' in result.request.body:  # sky Motion
                     names += ('dRA cos(Dec)', 'dDec')
                     units += ('arcsec/h', 'arcsec/h')
-                col_starts += (73, 81)
-                col_ends += (80, 89)
+                # Correct start and end for Motion columns if the results have 3 digit precision.
```

Comment:
I don't think this comment is needed, as the pull request documents the change.

Reply:
I have removed the comment line